ignite/contrib/handlers/base_logger.py

```diff
@@ -103,7 +103,26 @@ def _setup_output_metrics(self, engine: Engine) -> Dict[str, Any]:
                 output_dict = {"output": output_dict}
 
             metrics.update({name: value for name, value in output_dict.items()})
-        return metrics
+
+        metrics_dict = OrderedDict()
+
+        for name, value in metrics.items():
+            if value is None:
+                continue
+            if isinstance(value, numbers.Number):
+                metrics_dict[f"{self.tag}/{name}"] = value
+            elif isinstance(value, torch.Tensor) and value.ndimension() == 0:
+                metrics_dict[f"{self.tag}/{name}"] = value.item()
+            elif isinstance(value, torch.Tensor) and value.ndimension() == 1:
+                for i, v in enumerate(value):
+                    metrics_dict[f"{self.tag}/{name}/{i}"] = v.item()
+            else:
+                if isinstance(value, str) and log_text:
+                    metrics_dict[f"{self.tag}/{name}"] = value
+                else:
+                    warnings.warn(f"Logger output_handler can not log metrics value type {type(value)}")
```

Comment:
Here since we dont have the name of the logger in the base class I have raised a generic warning using the term "Logger". Would it be better to pass the name of the Logger to the base class and have that in the warning?

Reply:
I think we can use `self.__class__.__name__` instead of Logger.

Reply:
That won't be helpful as the name of the derived class is OutputHandler in all cases (not Logger name) right?

Reply:
OK, let's keep Logger.